actors/miner/src/lib.rs

```diff
@@ -3961,7 +3953,8 @@ fn process_early_terminations(
     notify_pledge_changed(rt, &pledge_delta)?;
 
     // Terminate deals.
-    let all_terminated = BitField::union(sectors_terminated.iter());
+    let all_terminated = BitField::try_from_bits(sectors_with_data)
```

Comment:
This should have a different name since it no longer includes all terminated sectors